packages/lit-helpers/README.md

```diff
@@ -120,3 +120,33 @@ html`
     }
   }
 </script>
+
+## Use of Custom Element classes instead tags into the DOM
```

Comment:
I would really love more documentation on this, what is it, why is it, what does it do, how does it do it